lib/src/upgrader.dart

```diff
@@ -239,6 +250,40 @@ class Upgrader {
     return true;
   }
 
+  /// Android info is fetched by parsing the html of the app store page.
+  Future<bool?> _getAndroidStoreVersion() async {
```

Comment:
Fetching the Android version from the Google Play Store should be in its own class and file. Use `ITunesSearchAPI` as an example. The call to fetch the version from the Google Play Store should happen after the appcast, just like `ITunesSearchAPI`. It should happen right after the call to `findCountryCode()`. The logic for this new code needs to align better with the existing logic in `_updateVersionInfo`.

Reply:
Okay, I will work on that and let you know when I finish.